src/js/ratinglist.js

```diff
@@ -39,13 +39,13 @@ function formatnames(names){
 	let a
 	for(a = 0;a < names.length;a++){
 		if(a === 0){
-			outarray.push(A({className:'firstname',href:"/games/search?playerw="+names[a]+"&mirror=on",target:'_blank'},names[a]))
+			outarray.push(A({className:'firstname',href:"/games?player_white="+names[a]+"&mirror=true",target:'_blank'},names[a]))
```

Comment:
Did you change the endpoint here in the other merge request?

Reply:
this is just a url to the games history UI with search params, not hitting the actual api endpoint until you get to /games